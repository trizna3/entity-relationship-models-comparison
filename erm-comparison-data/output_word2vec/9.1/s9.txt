transport model-EMPTY
airplane-airplanes
airplane model-EMPTY
marine-soldier
marine role-EMPTY
fight model-EMPTY
rocket type-rocket type
EXTRACT_ATTR_TO_OWN_ENTITY_SET:S-rocket type-fighter planes
