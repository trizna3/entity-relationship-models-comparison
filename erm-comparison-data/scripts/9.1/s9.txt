entity set airplanes {
	"serial number"
	"name"
};

entity set "model type" {
};

entity set "fighter planes" {
	"rocket type"
	"rocket count"
};

entity set "transport planes" {
	"person capacity"
	"material volume"
	"material weight"
};

entity set soldier {
	"name"
	"health"
	"maximal health"
	"speed"
	"strength"
	"agility"
	"inteligence"
};

entity set "soldier role" {
	"type"
	"health modifier"
	"maximal health modifier"
	"speed modifier"
	"strength modifier"
	"agility modifier"
	"inteligence modifier"
};

association "airplane model" between 
{
	airplanes MANY
	"model type" ONE
}{};

association "assigned to" between
{
	airplanes MANY
	soldier MANY
}{};

association "soldier role" between 
{
	"soldier role" ONE
	soldier MANY
}{};


generalization from "fighter planes" to "model type";

generalization from "transport planes" to "model type";
