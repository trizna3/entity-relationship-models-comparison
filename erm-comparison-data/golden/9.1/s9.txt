airplane model-model type
fight model-fighter planes
rocket type-rocket type
transport model-transport planes
airplane-airplanes
marine-soldier
marine role-soldier role
EXTRACT_ATTR_TO_OWN_ENTITY_SET:S-rocket type-fighter planes